### PR TITLE
Docs: dynamic subs on topics and wildcard behavior

### PR DESCRIPTION
**Pull request by jenmalloy**

### CODE REVIEW

File: documentation/modules/con-brokered-topic.adoc [hunk 1]
```diff
@@ -9,3 +9,18 @@
 The topic address type supports the publish-subscribe messaging pattern in which there are 1..N producers and 1..M consumers. Each message published to a topic address is forwarded to all subscribers for that address. A subscriber can also be durable, in which case messages are kept until the subscriber has acknowledged them.
 // !brokered.address.topic.longDescription:stop
 
+== Dynamic subscriptions on topics and wildcard behavior
+
+You can specify a dynamic subscription on topics in the brokered address space using wildcards. The wildcard behavior follows the MQTT syntax:
```
Comment (grs) [inline documentation/modules/con-brokered-topic.adoc]: I think I would avoid the term 'dynamic subscription' here if possible (I know I was the one who used it in discussion). I would say something like:

A client receiving from a topic address can specify a wildcard address with the topic address as the root. The wildcard behaviour follows...

Also maybe change the section title to just 'hierarchical topics and wildcards'

File: documentation/modules/con-standard-topic.adoc [hunk 2]
```diff
@@ -9,4 +9,19 @@
 The topic address type supports the publish-subscribe messaging pattern where there are 1..N producers and 1..M consumers. Each message published to a topic address is forwarded to all subscribers for that address. A subscriber can also be durable, in which case messages are kept until the subscriber has acknowledged them.
 // !standard.address.topic.longDescription:stop
 
+== Dynamic subscriptions on topics and wildcard behavior
+
+You can specify a dynamic subscription on topics in the standard address space using wildcards. The wildcard behavior follows the MQTT syntax:
```
Comment (grs) [inline documentation/modules/con-standard-topic.adoc]: As above, I would try and avoid the term dynamic subscription here.

File: documentation/modules/con-brokered-topic.adoc (hunk 1 above)
Comment (jenmalloy) [reply documentation/modules/con-brokered-topic.adoc]: @grs, excellent, thank you! Will fix

File: documentation/modules/con-standard-topic.adoc (hunk 2 above)
Comment (jenmalloy) [reply documentation/modules/con-standard-topic.adoc]: Done

**Pull request merged**